tests/pkg_littlefs/tests/01-run.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2017 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
```

Comment:
Flake8 reports that this one is unused, you can squash it directly.
https://travis-ci.org/RIOT-OS/RIOT/builds/538272513#L441

Reply:
Ah, should have noticed the warning locally. Fixed now